gunicorn/instrument/statsd.py

```diff
@@ -94,11 +94,16 @@ def access(self, resp, req, environ, request_time):
         Logger.access(self, resp, req, environ, request_time)
         duration_in_ms = request_time.seconds * 1000 + float(request_time.microseconds) / 10 ** 3
         status = resp.status
+        path = req.path.replace('/', '__')
```

Comment:
Since you replaced all `/`s with `__`s, `path` doesn't look like a correct variable name here.